### ISSUE TITLE
Calibration upload ea266f

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#466]
Calibration images were uploaded to the directory that starts with “`ea266f_`”.

Please read the [workflow description](https://github.com/lensfun/lensfun/blob/master/tools/calibration_webserver/workflow.rst) for further instructions about the calibration.

Additional comments by the uploader:

> Laowa 60mm 2.8 Ultra-Macro
> Sony A7R2
> 
> made vignetting shots for multiple distances. i think vignetting should be correctable below 0.24m. for 1m to infinity, this lens is probably uncorrectable. the integrated lenshood is generating a very strange vignetting effect for longer distances.

**Comment by mer-at** [comment]
i am the uploader

**Comment by junkyardsparkle** [comment]
@mer-at Wow, that's a lot of "pincushion"! Can you confirm that distortion is the same at macro distances? If not, it might be better to profile for that case, since it would seem to be the primary reason for using this lens.

**Comment by mer-at** [comment]
yeah it's pretty heavy even for macros.
here's an example of a 35mm negative on a lighttable in almost 1:1
![dsc03145](https://user-images.githubusercontent.com/11669553/52303048-42d79e80-298f-11e9-9d69-e5f799d0fdc9.jpg)

**Comment by junkyardsparkle** [comment]
Ok, if the infinity correction doesn't fit that image well, I'll just create two profiles, normal and "close-up", since this lens will need to be manually selected in software anyway... That film shot is a full, uncropped frame, correct?

**Comment by mer-at** [comment]
Yes that picture is uncropped.

**Comment by junkyardsparkle** [comment]
Well, this one was interesting. I thought I might need to discard the vignetting shots at infinity, due to the rectangualr shading (lensfun only understands radial fall-off), but the results of applying the radial correction don't look as bad as I thought they would, so I kept them. If it's desired to use subtler correction that doesn't address the lens hood, a closer-than-actual distance can be manually selected... I did notice that the shortest distance offered by darktable is 0.25, though... this might be worth a bug report.

Anyway, thanks for the comprehensive set of images, and sorry about the long wait.
